platform/darwin/src/MGLCircleStyleLayer.h

```diff
@@ -153,20 +153,32 @@ typedef NS_ENUM(NSUInteger, MGLCircleTranslateAnchor) {
  `NSValue` object containing a `CGVector` struct set to 0 points from the left
  and 0 points from the top. Set this property to `nil` to reset it to the
  default value.
+ 
+ This attribute corresponds to the <a
+ href="https://www.mapbox.com/mapbox-gl-style-spec/#layout-circle-circle-translate"><code>circle-translate</code></a>
```

Comment:
This URL leads to the root of the style spec. Isn't the correct ref `#paint-circle-translate`?